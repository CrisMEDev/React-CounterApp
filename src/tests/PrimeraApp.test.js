// import {render} from '@testing-library/react'

import { shallow } from 'enzyme';

import PrimeraApp2 from '../PrimeraApp';

describe('Pruebas en <PrimeraApp />', () => {

    // test('Debe de mostrar el mensaje "Hola, Soy Cristian ME"', () => {
        
    //     const saludo = 'Hola, Soy Cristian ME';

    //     const wrapper = render( <PrimeraApp2 saludo={saludo} numero={0} /> );

    //     // Concatener !!! hace que la prueba falle pero es dificil de leer el error por eso se usar√° enzyme
    //     // expect(wrapper.getByText( saludo + '!!!' )).toBeInTheDocument();

    //     expect(wrapper.getByText( saludo )).toBeInTheDocument();

    // });
    
    test('Debe de mostrar <PrimeraApp /> correctamente', () => {

        const numero = 0;
        const saludo = 'Hola, Soy Cristian ME';
        const wrapper = shallow( <PrimeraApp2 saludo={saludo} numero={numero} /> );

        expect( wrapper ).toMatchSnapshot();
        
    })
    

});



