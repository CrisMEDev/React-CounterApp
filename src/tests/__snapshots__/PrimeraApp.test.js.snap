// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> Debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, Soy Cristian ME
  </h1>
  <h2>
    0
  </h2>
  <pre>
    {
   "nombre": "Cristian",
   "edad": 26
}
  </pre>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
